vm_insnhelper.c

```diff
@@ -5550,6 +5550,13 @@ vm_get_special_object(const VALUE *const reg_ep,
     }
 }
 
+// ZJIT implementation is using the C function
+// and needs to call a non-static function
+VALUE rb_vm_get_special_object(const VALUE *reg_ep, enum vm_special_object_type type)
```

Comment:
I'm not sure if exposing static VM functions for jit is allowed as I didn't see many examples. But we seems to do `rb_vm_concat_array` for YJIT?

Reply:
This is allowed